main.go

```diff
@@ -6,162 +6,59 @@ import (
 	"io/ioutil"
 	"log"
 	"net/http"
-	"net/http/cookiejar"
-	"net/url"
-	"strconv"
-	"time"
 )
 
 const (
-	// APIUrl is the deezer API
+	// APIUrl is the hidden Deezer API url
 	APIUrl = "http://www.deezer.com/ajax/gw-light.php"
-	// LoginURL is the API for deezer login
-	LoginURL = "https://www.deezer.com/ajax/action.php"
-	// deezer domain for cookie check
-	Domain = "https://www.deezer.com"
+	// Mobile api url, used for loginless download
+	MobileUrl = "https://api.deezer.com/1.0/gateway.php"
+	// Mobile api key
+	ApiKey = "4VCYIJUCDLOUELGD1V8WBVYBNVDYOXEWSLLZDONGBBDFVXTZJRXPR29JRLQFO6ZE"
```

Comment:
I'm sorry but i want some clarification here, that if i use the Mobile api url, i don't need to provide any User Token at all?

Reply:
No, the CSRF token is not needed when using the mobile api. The mobile api does need a sid token though, which GetTokens() now provides.